### ISSUE TITLE
Preprocessing test set

### ISSUE THREAD

**Issue by matejMartinc** [honghanhh/terminology-extraction#7]
Hey Hahn,

Went through the code and found one thing worth checking :) . In preprocessing.py (in line 89) you remove the sentences that do not have terms for the train set, right? Do you do the same for the test set on which you apply the trained model? If so, I imagine the classifier would have an easier job of only predicting terms for sentences containing terms? Predictions for sentences that have no terms inside would on the other hand probably produce some false positives that would be added to the final list and decrease overall precision?

**Comment by honghanhh** [comment]
Hi @matejMartinc,
As discussed in our meeting, I remove the sentences that do not have terms for only the training set (corp, equi, wind) as the result in folder processed_data/ and process_nes_data/ and predict on all the text in htfl domain (the preprocess.py does not apply for test set). Therefore, I don't think it affect the prediction results.

**Comment by matejMartinc** [comment]
Great, tnx!
